tempto-examples/src/main/java/com/teradata/tempto/examples/TemptoExamples.java

```diff
@@ -21,13 +21,12 @@
 
 public class TemptoExamples
 {
-
     public static void main(String[] args)
     {
         TemptoRunnerCommandLineParser parser = TemptoRunnerCommandLineParser
                 .builder("tempto examples")
                 .setTestsPackage("com.teradata.tempto.examples", false)
-                .setConfigFile(DEFAULT_TEST_CONFIGURATION_LOCATION, false)
+                .setConfigFile(DEFAULT_TEST_CONFIGURATION_LOCATION, true)
```

Comment:
related. I change global configuration to test configuration without table_manager or without databases at all